content/articles/why-unikernels-may-soon-replace-docker/index.md

```diff
@@ -0,0 +1,73 @@
+
+The fact remains that containers have positively evolved the cloud technology world, but they are not yet what [DevOps](https://azure.microsoft.com/en-us/overview/what-is-devops/#culture) requires.  Conversely, unikernels are a new software infrastructure paradigm concerned to realize the DevOps dreams. They provide specialized, single-address space machine images.
+
+A unikernel consists of the below components:
+- System libraries
+- Language runtime
+- Necessary applications
+
+These components are built together to form one bootable [Virtual Machine](https://www.vmware.com/topics/glossary/content/virtual-machine) (VM) image that can execute on a [hypervisor](https://www.vmware.com/topics/glossary/content/hypervisor).
+
+### Comparison between Containers and Unikernels
+The critical aspect of comparing the two is that containers still differentiate between operators responsible for managing the applications and the developers who code the application.
+
+In contrast, unikernels strive to meet the DevOps requirements by doing away with the above barrier and ensuring that the development team bundles everything needed into the deployment unit.
+
+Furthermore, deploying unikernel applications requires rebuilding the VM again using configuration tools such as [Terraform](https://www.terraform.io/intro/index.html) and [Jenkins](https://www.jenkins.io/doc/book/).
+
+### Reasons why Unikernels may replace containers
+**Unikernels avoid vendor-lock-ins**
+
+The main advantage of using unikernels in the organization's infrastructure is added agility and avoiding vendor lock-in. A unikernel is a VM at its core; it also supports multiple cloud providers and platform-independent.
+
+Furthermore, the basic unikernel unit of computing is a VM, which means that it is easy to shuffle between different cloud platforms. 
+
+Unikernels adopts the [serverless](https://www.redhat.com/en/topics/cloud-native-apps/what-is-serverless) approach. Of course, the term serverless does not mean there is no server, but most of the time, it needs minimal support and oversight by the operations team.
+
+**Containers are still too complex**
+
+Although containers have solved many problems in the DevOps space, containers as paradigms were not what DevOps would want the future to be.
+
+According to the complexity involved when operating containers is often very time-consuming for the teams. Also, the teams often find out that the tools are not as perfect as expected and often cause issues in production,
+
+Most of the time requires the whole operations team to assist in managing and deploying a Kubernetes-based application. On the other hand, to deploy a unikernel to [Google Cloud](https://cloud.google.com/) nowadays via Ops is a matter of few commands, and it will be up and running within minutes.
```

Comment:
```suggestion
Operating containers may require the whole operations team to assist in managing and deploying a Kubernetes-based application. On the other hand, to deploy a unikernel to [Google Cloud](https://cloud.google.com/) nowadays via Ops is a matter of few commands, and it will be up and running within minutes.